_i18n/pt_BR/factors/prefer-local-over-remote.md

```diff
@@ -0,0 +1,22 @@
+Nos últimos anos, alguns times de desenvolvimento começaram a usar abordagens que requerem menos trabalho de desenvolvimento, pagando o custo da experiência de uso do aplicativo ao mover mais lógica para um backend remoto e ter o aplicativo iOS sendo apenas um cliente simples que mostra resultados do servidor. Esse approach resulta na frustração de usuários quando o aplicativo é utilizado em situações com uma conexão de internet não tão perfeitas assim (como em metrôs, elevadores ou uma rede WiFi irregular).
```

Comment:
Maybe `que requerem menos trabalho de desenvolvimento, às custas da experiência de uso do aplicativo, ao mover mais...` instead of `que requerem menos trabalho de desenvolvimento, pagando o custo da experiência de uso do aplicativo ao mover mais...`?